content/manager_architecture/metrics-flow.md

```diff
@@ -21,30 +21,40 @@ You can send metrics to the management environment using any transport (agent),
 
 RabbitMQ stores metrics within a metrics-dedicated, non-durable, non-exclusive topic exchange.
 
-After a metric is consumed it is removed from the queue. In principle, you can consume metrics directly from RabbitMQ for processing in systems outside Cloudify. Cloudify does not provide any implementation to officially support this, however it is enables by the architecture, and by removing the proprietary consumer, you it is possible to consume metrics directly from RabbitMQ.
+After a metric is consumed it is removed from the queue. In principle, you can consume metrics directly from RabbitMQ for processing in systems outside Cloudify. Cloudify does not provide any implementation to officially support this, however it is enables by the architecture, and by removing the proprietary consumer, it is possible to consume metrics directly from RabbitMQ.
+
+RabbitMQ connection is ssl secured and each deployments has its own vhost, hence connection parameters are the following:
```

Comment:
The RabbitMQ connection is secured by SSL and each deployments has its own vhost. The connection parameters are: